profile/README.md

```diff
@@ -1,7 +1,13 @@
-[![eProsima](https://raw.githubusercontent.com/eProsima/.github/main/profile/eprosima_github_banner.png)](https://eprosima.com/)
+# [![eProsima](https://raw.githubusercontent.com/eProsima/.github/main/profile/eprosima_github_banner.png)](https://eprosima.com/)
 
-[eProsima](https://eprosima.com/) provides networking middleware solutions, open source and available to everyone, to empower communications around the globe. Our high-quality open-source DDS products have been chosen for robotics, automated technology, and IoT. eProsima Fast DDS has been adopted by ROS 2 - the main framework for robotics development - and eProsima Micro XRCE-DDS is the base of micro-ROS, the official ROS 2 extension for microcontrollers.
+[eProsima](https://eprosima.com/) 
```

Comment:
It looks a little bit empty or not finished the eProsima word with the link here.